src/shared/schema.ts

```diff
@@ -139,7 +139,10 @@ const spoilerNodeSpec: NodeSpec = {
     toDOM(node) {
         return [
             "blockquote",
-            { class: "spoiler" + (node.attrs.revealed ? " is-visible" : "") },
+            {
+                "class": "spoiler" + (node.attrs.revealed ? " is-visible" : ""),
+                "data-spoiler": "Reveal spoiler",
```

Comment:
For my sanity later, when implementing localization

```suggestion
                // TODO localization
                "data-spoiler": "Reveal spoiler",
```